README.md

```diff
@@ -49,6 +51,26 @@ the image first.
 
 - When confirmed, click on the `Flash` button to start flashing procedure.
 
+### bmaptool
+
+If you prefer a command-line tool, then you should use bmaptool, just install it with
+
+```
+sudo apt install bmap-tools
+```
+
```

Comment:
@WiktorG351 firstly, let's point to gh releases https://github.com/3mdeb/meta-rte/releases to get the image (wic.gz and wic.bmap files) and then show commad to flash it

Reply:
fixed in https://github.com/3mdeb/meta-rte/blob/734d074babc8652f7e6ba7f7ada1e5a296b60f0b/README.md?plain=1#L80